collectors/mesos/agent/agent.go

```diff
@@ -146,12 +147,17 @@ func getFrameworkInfoByFrameworkID(frameworkID string, frameworks []frameworkInf
 func getLabelsByContainerID(containerID string, frameworks []frameworkInfo, log *logrus.Entry) map[string]string {
 	labels := map[string]string{}
 	for _, framework := range frameworks {
-		log.Debugf("Attemping to add labels to %v framework", framework)
+		log.Debugf("Attempting to add labels to %v framework", framework)
 		for _, executor := range framework.Executors {
 			log.Debugf("Found executor %v for framework %v", framework, executor)
 			if executor.Container == containerID {
 				log.Debugf("ContainerID %v for executor %v is a match, adding labels", containerID, executor)
 				for _, pair := range executor.Labels {
+					if len(pair.Value) > maxLabelLength {
+						log.Warnf("Label %s is longer than %d chars; discarding label", maxLabelLength, pair.Key)
+						log.Debugf("Discarded label value: %s", pair.Value)
```

Comment:
I actually think it's really useful to see the actual label that caused the warning when you're debugging. I know it's a little funky to warn and debug one after the other. I guess the user could follow the other debug traces to identify the issue if they had to. What's your feeling?